core/Http.php

```diff
@@ -22,7 +22,7 @@ class Http
     /**
      * Returns the "best" available transport method for {@link sendHttpRequest()} calls.
      *
-     * @return string Either `'curl'`, `'fopen'` or `'socket'`.
+     * @return string|null Either curl, fopen, socket or null of no method is supported.
```

Comment:
Not really related to the code review, but is there a reason you removed the backticks & quotes?